lib/bitters/generator.rb

```diff
@@ -76,5 +85,27 @@ def stylesheets_directory
     def top_level_directory
       File.dirname(File.dirname(File.dirname(__FILE__)))
     end
+
+    def command_exists?(command)
+      `which #{command}` != ""
+    end
+
+    def change_file_ext(file, ext)
+      no_ext = File.join(File.dirname(file), File.basename(file, ".*"))
+      "#{no_ext}.#{ext}"
+    end
+
+    def convert_syntax_to_sass
+      if command_exists?("sass-convert")
+        Dir.glob(File.join(install_path, "**/*.scss")).each do |file|
+          from = file.to_s
+          to = change_file_ext(file, "sass")
+          `sass-convert #{from} #{to}`
```

Comment:
Was this working for you without telling it what syntax to convert to and from? According to `sass-convert --help`, it would need to be done like this:

```
`sass-convert --from scss --to sass #{from} #{to}`
```